pkg/broker/broker.go

```diff
@@ -660,7 +663,7 @@ func (a AnsibleBroker) Provision(instanceUUID uuid.UUID, req *ProvisionRequest,
 
 // Deprovision - will deprovision a service.
 func (a AnsibleBroker) Deprovision(
-	instance apb.ServiceInstance, planID string, skipApbExecution bool, async bool,
+	instance apb.ServiceInstance, planID string, skipApbExecution bool, async bool, userInfo UserInfo,
```

Comment:
I don't see the update of the `parameters[requestingUserKey] = getRequestingUser(userInfo)` for Deprovision or Unbind or Update. 

I think we need to do this be a separate user could request those actions and if we want to alert the APBs of the users that are interacting with them, I think that we should update it. I wonder if we should re-name this field then to `_apb_last_requesting_user`?

Reply:
I have updated as you advised, to override the userInfo in the Deprovision, Unbind, and Update